src/components/TodosFilter/TodosFilter.tsx

```diff
@@ -0,0 +1,70 @@
+import React, { useContext } from 'react';
+import classNames from 'classnames';
+import { TodosContext } from '../../TodosContext';
+import { Status } from '../../types';
+
+export const TodosFilter: React.FC = () => {
+  const {
+    incompletedCount,
+    areSomeCompleted,
+    removeCompleted,
+    status,
+    setStatus,
+  } = useContext(TodosContext);
+
+  return (
+    <footer className="footer" data-cy="todosFilter">
+      <span className="todo-count" data-cy="todosCounter">
+        {`${incompletedCount} items left`}
+      </span>
+
+      <ul className="filters">
+        <li>
```

Comment:
can you check some similar things in this link items, create some array, and iterate through it?
like
```
const VALUES = [
  { path: '#/', filterValue: Status.All, label: 'All' },
  { path: '#/active', filterValue: Status.Active, label: 'Active' },
  { path: '#/completed', filterValue: Status.Completed, label: 'Completed' },
]


jsx
{VALUES.map(item => (
  <li key={item.path}>
    <a
      href={item.path}
      className={classNames({
        selected: status === item.filterValue,
      })}
      onClick={() => setStatus(item.filterValue)}
    >
      {item.label}
    </a>
  </li>
))}
```